playground/src/db/models/user.ts

```diff
@@ -0,0 +1,18 @@
+import {
+  Field,
+  PrimaryKey,
+  TigrisCollection,
+  TigrisDataTypes,
+} from "@tigrisdata/core";
+
+@TigrisCollection("user")
+export class User {
+  @PrimaryKey(TigrisDataTypes.INT64, { order: 1, autoGenerate: true })
+  id?: bigint;
```

Comment:
if bigint support is required in tigris - it needs to have the config option `supportBigInt` set to `true`.

Reply:
Hmm, it seems to work fine without the supportBigInt flag.